simulation/test_tracker.cpp

```diff
@@ -262,6 +262,8 @@ TORRENT_TEST(announce_interval_1200)
 
 struct sim_config : sim::default_config
 {
+	explicit sim_config(bool ipv6 = true) : ipv6(ipv6) {}
```

Comment:
actually, I just realized this is local to this test, it probably doesn't matter